frontend/src/settings.tsx

```diff
@@ -731,14 +731,20 @@ export const POSSIBLE_TRANSFORMS: PossibleTransform[] = [
     display_name: "Resize",
     object_name: "transforms.Resize",
     parameters: {
-      size: {
+      height: {
         index: 0,
-        parameter_name: "(H, W)",
+        parameter_name: "height",
         min: 1,
         max: 1000,
-        default: "(32, 32)",
-        parameter_type: "tuple",
-      } as LayerParameter<"tuple">,
+        parameter_type: "number",
+      } as LayerParameter<"number">,
```

Comment:
same situation as the constants file @farisdurrani @karkir0003

Reply:
not too sure on this as well

Reply:
Same answer as the constants